SPEC.md

```diff
@@ -275,6 +275,14 @@ a response.
 Format of this header value is number + unit, where unit can be `ms` for milliseconds, `s` for seconds, and `m` for
 minutes.
 
+### `Operation-Timeout`
```

Comment:
TIL about https://http.dev/prefer. We could probably use that if we wanted (though don't think we can reuse `wait` as defined at https://www.rfc-editor.org/rfc/rfc7240.html#section-4.3 because it is the same as request timeout).